dali_tf_plugin/setup.py.in

```diff
@@ -79,12 +79,13 @@ For more details please check the
       packages=find_namespace_packages(include=['nvidia.*']),
       include_package_data=True,
       zip_safe=False,
-      python_requires='>=3.10, <3.14',
+      python_requires='>=3.10, <3.15',
```

Comment:
I'm wondering, if we really need the upper-bound restriction on Python version. Do you recall what was the benefit of having it here?